frontend/awx/views/schedules/hooks/scheduleHelpers.tsx

```diff
@@ -77,6 +77,48 @@ export function useGetSchedulCreateUrl(sublistEndPoint?: string) {
   return createUrl;
 }
 
+export function useGetSchedulEditUrl(sublistEndPoint?: string) {
```

Comment:
Rather than creating a whole new function can we reuse the `useGetSchedulCreateUrl` function inside scheduleHelpers.tsx?  If so, I'd recommend a name change for the function as well.

Reply:
We can't use the useGetSchedulCreateUrl because they call different routes, create and edit